src/main/java/com/ssbb/kanban/controller/project/ProjectController.java

```diff
@@ -26,9 +24,25 @@
 
 	@RequestMapping(value = "project", method = RequestMethod.GET)
 	public String loadProject(ModelMap map, HttpSession session) {

```

Comment:
There is no longer any logic here so I think we can remove this method and allow our two methods below to directly load the "project" page.